### DISCUSSION TITLE
Rename model with relations

### DISCUSSION THREAD

**Discussion by rnwcode** [MichalLytek/typegraphql-prisma#117] [Q&A]
Hey, 
I wanted to try to rename a model which is also used by others. 

```
/// @@TypeGraphQL.type(name: "DevTeet") 
model Test {
...
} 
```

However, when I try to create a schema from it I get the following error: 

```
prisma/generated/type-graphql/resolvers/crud/DevTest/DevTestCrudResolver.ts:154:34 - error TS2552: Cannot find name 'AggregateDevDevTest'. Did you mean 'AggregateDevTest'?

@TypeGraphQL.Query(_returns => AggregateDevDevTest, {
```

Is this generally not possible or is it a bug?

**Comment by MichalLytek** [comment]
Looks like a bug, I need to investigate it 👀

**Reply by MichalLytek** [reply]
Please create an issue with reproduction repo.
